apps/mobile/convex/confect/http-api.ts

```
@@ -0,0 +1,150 @@
+  .add(MessagesApiGroup)
+  .add(SessionsApiGroup)
+  .prefix("/api/v1") {}
+
+// Implementation of API groups
+const UsersApiGroupLive = HttpApiBuilder.group(OpenAgentsApi, "users", (handlers) =>
+  handlers
+    .handle("getCurrentUser", () =>
+      Effect.gen(function* () {
+        // TODO: Fix HTTP API integration with Confect functions
+        return Option.none();
+      })
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)